model/DataStore/PersistDataService.php

```diff
@@ -124,7 +124,7 @@ private function persistArchive(ResourceSyncDTO $resourceSyncDTO): void
 
             $this->moveExportedZipTest($folder, $resourceSyncDTO->getResourceId(), $resourceSyncDTO);
         } finally {
-            $tempDir->removeDirectory($folder);
+            //FIXME DO not remove directory for tests $tempDir->removeDirectory($folder);
```

Comment:
Why we commented this?

Reply:
It is explained in the comment, I was testing it locally and I did not want the tmp file to be removed. 

Removed from the PR.

Thanks!